pkgs/top-level/all-packages.nix

```diff
@@ -17730,6 +17730,8 @@ in
 
   das_watchdog = callPackage ../tools/system/das_watchdog { };
 
+  komodo-python-dbgp = callPackage ../development/tools/komodo-python-dbgp {};
```

Comment:
`all-packages.nix` are *not strictly* alphabetized, but could you try keeping that at least in some way?